README.md

```diff
@@ -1,107 +1,77 @@
-did-io
-=================
+# DID Whisper Client _(did-whisper)_
 
 [![Build Status](https://travis-ci.org/digitalbazaar/did-io.png?branch=master)](https://travis-ci.org/digitalbazaar/did-io)
 
-A Decentralized Identifier resolution library for the browser and
-node.js.
+> A [DID](https://w3c-ccg.github.io/did-spec/) (Decentralized Identifier) resolution library for Javascript
 
-Install
--------
+## Table of Contents
 
-The did-io library is available through npm for both node.js and 
-the browser.
+- [Security](#security)
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [Commercial Support](#commercial-support)
+- [License](#license)
 
-## Requiring did-io
+## Security
 
-### node.js + npm
+TBD
 
-```
-npm install did-io
-```
-
-```js
-var didio = require('did-io');
-```
-
-### JSPM
+## Background
 
-```
-jspm install npm:did-io
-```
-
-``` js
-import * as didio from 'did-io';
-```
+TBD
 
+See also (related specs):
 
-Quick Examples
---------------
+* [Decentralized Identifiers (DIDs) - Data Model and Syntaxes](https://w3c-ccg.github.io/did-spec/)
+* [Veres One DID Method](https://w3c-ccg.github.io/didm-veres-one/)
+* [Web Ledger Protocol](https://w3c.github.io/web-ledger/)
+* [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/)
+* [Linked Data Proofs](https://w3c-dvcg.github.io/ld-proofs/)
+* [Object Capabilities for Linked Data](https://w3c-ccg.github.io/ocap-ld/)
 
-Retrieve a DID document:
+## Install
```

Comment:
Updated version will be in npm at some point so we want to keep the "npm install did-io" instructions too.

Reply:
Changed.